docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Elytron_OIDC_Client.adoc

```diff
@@ -173,6 +173,195 @@ It is also possible to configure the scope values that would be used via the sco
 
 NOTE: If both the deployment and subsystem configuration is present, then the subsystem configuration will override the deployment values.
 
+== Sending Request Objects as a JWT
+
+NOTE: Since this attribute is a preview level feature, the WildFly server's stability level must be set to `preview`
```

Comment:
"Since this attribute..."

Might be good to reword this a bit. Since this is the first sentence, it's not clear what's being referred to here.